packages/ui-themes/src/index.ts

```diff
@@ -22,23 +22,17 @@
  * SOFTWARE.
  */
 
-import {
-  theme as canvasHighContrast,
-  CanvasHighContrastTheme
-} from '@instructure/canvas-high-contrast-theme'
-
-import {
-  theme as canvas,
+import type { CanvasHighContrastTheme } from '@instructure/canvas-high-contrast-theme'
+import type {
   CanvasTheme,
   CanvasBrandVariables
 } from '@instructure/canvas-theme'
+import type { InstructureTheme } from '@instructure/instructure-theme'
+import type { BaseTheme } from '@instructure/shared-types'
 
-import {
-  theme as instructure,
-  InstructureTheme
-} from '@instructure/instructure-theme'
-
-import { BaseTheme } from '@instructure/shared-types'
+import { theme as canvasHighContrast } from '@instructure/canvas-high-contrast-theme'
+import { theme as canvas } from '@instructure/canvas-theme'
+import { theme as instructure } from '@instructure/instructure-theme'
```

Comment:
updating ESLint caused this issue to pop up, could it be that it somehow activated https://www.npmjs.com/package/eslint-plugin-no-explicit-type-exports ?